packages/boot/tools/authorityViz.js

```diff
@@ -36,7 +38,8 @@ const styles = {
  * }} GraphNode
  */
 function* fmtGraph(nodes, neighbors) {
-  const q = txt => JSON.stringify(txt.replace(/\./g, '_'));
+  // TODO Why is this distinct from `q`?
```

Comment:
Why was this distict from `q`? Does it still need to be distinct?

Reply:
I think this is sufficiently distinct from `assert.quote` that it should be left as-is. The assert machinery is not in use here, just borrowing the meme of “`q` means `quote`” and applying it for graphviz dot format.

Reply:
Thanks. I removed the TODO comment and renamed `q2` to `quote`. Done.